doc/getting_started/index.rst

```diff
@@ -71,7 +74,7 @@ Next, clone the Zephyr source code repositories from GitHub using the
 
 .. code-block:: console
 
-   west init zephyrproject
+   west init zephyrproject --mr v1.14.1
```

Comment:
You likely know best, but since this is being applied to the 1.14 branch, should this be ``-mr v1.14.0`` here, and then a separate PR for applying it to the 1.14.1 branch?  Othewise someone pulling the 1.14.0 branch docs will get incorrect instructions, no?

Reply:
There is no 1.14.1 branch @dbkinder. The `v1.14-branch` is the basis for all 1.14.x releases

Reply:
Interesting, OK. Thanks for clarifying.

Reply:
the above will only work when we have v1.14.1 tag, so the documentation will be inaccurate until it is published with the 1.14.1 release. Guess that is ok.

Reply:
I wasn't aware we were building 1.14 branch documentation? I don't see that in a document release version. Developers building v1.14-branch docs for themselves I guess will have to know what they are doing.